src/APIInterfaces/BackendAPI/Requests/CourseRequests.ts

```diff
@@ -18,7 +18,7 @@ const COURSE_QUESTION_INSTANCE_PATH = url.resolve(COURSE_QUESTION_GRADE_PATH, 'i
 const COURSE_QUESTIONS_PATH = url.resolve(COURSE_PATH, 'questions/');
 const COURSE_DEF_PATH = url.resolve(COURSE_PATH, 'def/');
 const COURSE_ENROLL_PATH = url.resolve(COURSE_PATH, 'enroll/');
-const COURSE_ASSESS_PATH = url.resolve(COURSE_PATH, 'assessment/');
+const COURSE_ASSESS_PATH = url.resolve(COURSE_PATH, 'assessment/topic/');
```

Comment:
On dev there's a COURSE_ASSESS_PATH you can use.

Reply:
Oh lol, you're changing it 🤦

Reply:
no worries - I adjusted for it (I want to change that backend path anyhow now that this isn't strictly for assessments)

Reply:
We should have another constant joining topic back to it?